### DISCUSSION TITLE
Addition of openssf scorecard badge?

### DISCUSSION THREAD

**Discussion by andy778** [CycloneDX/cyclonedx-python#708] [Ideas]
If one is the owner its quite easy to add the [scorecard](https://securityscorecards.dev/) badge, and you have already "OpenSSF Best Practices".

Reason is to ensure that Basic things like branch protection, to reduce some of the tings that one don't want in the repos...

**Comment by jkowalleck** [comment]
see https://github.com/CycloneDX/cyclonedx-python/pull/573
